input/groups/back.json

```diff
@@ -4,78 +4,85 @@
       "Starter": [
         2340
       ],
-      "Crafting: Armorsmith": [
+      "Crafting Discipline: Armorsmith": [
         2304,
         2305,
         2308,
         2279,
         2301,
         2303
       ],
-      "Crafting: Leatherworker": [
+      "Crafting Discipline: Leatherworker": [
         2292,
         2293,
         2294,
         2295,
         2296,
         2315
       ],
-      "Crafting: Tailor": [
+      "Crafting Discipline: Tailor": [
         2307,
         2309,
         2282,
         2283,
         2316,
         2286
       ],
-      "Crafting: Artificer": [
+      "Crafting Discipline: Artificer": [
         2320,
         2297,
         2298,
         2299,
         2284,
         2300
       ],
-      "Crafting: Huntsman": [
+      "Crafting Discipline: Huntsman": [
         2321,
         2322,
         2323,
         2310,
         2311,
         2312
       ],
-      "Crafting: Weaponsmith": [
+      "Crafting Discipline: Weaponsmith": [
         2288,
         2290,
         2306,
         2291,
         2317,
         2287
       ],
-      "Crafting: Chef": [
+      "Crafting Discipline: Chef": [
         2289,
         2324,
         2277,
         2314,
         2302,
         2319
       ],
-      "Crafting: Jeweler": [
+      "Crafting Discipline: Jeweler": [
         2276,
         2278,
         2280,
         2281,
         2313,
         2285
       ],
-      "Crafting: Scribe": [
+      "Crafting Discipline: Scribe": [
```

Comment:
I think I'll sort these alphabetically by discipline while I'm at it.

Reply:
The current sorting is armor/weapon/other, which is neat since it also happens to leave Scribe at the bottom, which was the last one to be released.

Nevermind, just realized Scribe shouldn't be there, since it requires Heart of Thorns lmao.

Reply:
Ah, true. :)

Reply:
I've changed it myself to alphabetical as well, less trouble in general.